<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YolcuGirisFormu</class>
 <widget class="QWidget" name="YolcuGirisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btnKaydet {
	qproperty-icon: url(:/Resimler/Kaydet.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;&amp;Kaydet&quot;;
   qproperty-default: true;
}

QPushButton#btnIptal {
	qproperty-icon: url(:/Resimler/Iptal.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;İ&amp;ptal&quot;;
}

QLabel {
	font: 700 12pt &quot;Roboto&quot;;
}

QLabel#lblBaslik {
	font: 700 italic 18pt &quot;Roboto&quot;;
}

QLabel#lblSimge {
	qproperty-maximumSize: 48px 48px;
}

QLineEdit {
	font: 12pt &quot;Roboto&quot;;
}
QSpinBox {
	font: 12pt &quot;Roboto&quot;;
}

QPlainTextEdit {
	font: 12pt &quot;Roboto&quot;;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSimge">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../Kaynaklar.qrc">:/Resimler/YeniYolcu</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBaslik">
       <property name="text">
        <string>Yeni Yolcu Girişi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblYolcuAd">
         <property name="text">
          <string>Ad</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leYolcuAd"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblYolcuSoyad">
         <property name="text">
          <string>Soyad</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leYolcuSoyad"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblYas">
         <property name="text">
          <string>Yas</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spnYas">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblHesKodu">
         <property name="text">
          <string>HES Kodu</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leHesKodu"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblTcKimlikNo">
         <property name="text">
          <string>TC Kimlik NO</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leTcKimlikNo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnKaydet">
         <property name="text">
          <string>&amp;Kaydet</string>
         </property>
         <property name="icon">
          <iconset resource="../../Kaynaklar.qrc">
           <normaloff>:/Resimler/Kaydet.png</normaloff>:/Resimler/Kaydet.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnIptal">
         <property name="text">
          <string>İ&amp;ptal</string>
         </property>
         <property name="icon">
          <iconset resource="../../Kaynaklar.qrc">
           <normaloff>:/Resimler/IptalEt</normaloff>:/Resimler/IptalEt</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections/>
</ui>
